subprojects/test-runner/test-report-jvm/src/main/kotlin/com/avito/test/report/arguments/PropertiesArgsProvider.kt

```diff
@@ -0,0 +1,24 @@
+package com.avito.test.report.arguments
+
+import com.avito.android.test.report.arguments.ArgsProvider
+import java.io.Serializable
+import java.util.Properties
+
+public class PropertiesArgsProvider(private val properties: Properties) : ArgsProvider {
+
+    override fun getOptionalArgument(name: String): String? {
+        return properties.getProperty(name, "").let { it.ifBlank { null } }
```

Comment:
It will return null if the property is an empty string.
We can check the existence of property and don't make such wrong assumptions.